keras/utils/generic_utils.py

```diff
@@ -532,3 +532,51 @@ def slice_arrays(arrays, start=None, stop=None):
             return arrays[start:stop]
         else:
             return [None]
+
+
+def to_data_format(values, data_format, skip=1):
```

Comment:
May I suggest the following signature:

```
pattern = [batch_size, steps, height, width, dimensions]
transposed_shape = transpose_shape(shape, target_format=data_format, spatial_axes=(2, 3))
```

I think that would be more readable and explicit than "to_data_format" and "skip".

Also, it would be more readable to explicitly pass keyword arguments when calling the function, e.g. 

`transpose_shape(shape, data_format=data_format, spatial_axes=(2, 3))
`
rather than
`transpose_to_data_format(shape, data_format, (2, 3))`